scripts/deploy_staging.ts

```diff
@@ -45,6 +47,7 @@ const UpkeepParams = {
 };
 
 const FAKE_VOW_ADDRESS = "0xA950524441892A31ebddF91d3cEEFa04Bf454466";
+const UNISWAP_V3_FACTORY_ADDRESS = "0x1F98431c8aD98523631AE4a59f267346ea31F984";
```

Comment:
This is configurable per environment and should be moved to a staging prefixed env variable?

Reply:
All addresses are the same in every environment. But I think that moving it to an env variable might be good as we might use that factory addresses in other places?

Reply:
That's right but the rest of the config is there. It will also work with new uniswap deployments.